README.md

```diff
@@ -40,25 +40,52 @@ Installing `paprika` from the `conda-forge` channel can be achieved by adding `c
 
 ```
 conda config --add channels conda-forge
+conda config --set channel_priority strict
 ```
 
-Once the `conda-forge` channel has been enabled, `paprika` can be installed with:
+Once the `conda-forge` channel has been enabled, `paprika` can be installed with `conda`:
 
 ```
 conda install paprika
 ```
 
-It is possible to list all the versions of `paprika` available on your platform with:
+or with `mamba`:
+
+```
+mamba install paprika
+```
+
+It is possible to list all of the versions of `paprika` available on your platform with `conda`:
 
 ```
 conda search paprika --channel conda-forge
 ```
 
+or with `mamba`:
+
+```
+mamba search paprika --channel conda-forge
+```
+
+Alternatively, `mamba repoquery` may provide more information:
+
+```
+# Search all versions available on your platform:
+mamba repoquery search paprika --channel conda-forge
+
+# List packages depending on `paprika`:
+mamba repoquery whoneeds paprika --channel conda-forge
+
+# List dependencies of `paprika`:
+mamba repoquery depends paprika --channel conda-forge
+```
+
 
 About conda-forge
 =================
 
-[![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](http://numfocus.org)
+[![Powered by
+NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org)
```

Comment:
I'm guessing the `NumFOCUS` is on a new line because of linting, right?

Reply:
Right, this will be rendered as a badge in the README so the line spacing should not matter.